src/Presto/Backend/Language/Flow.purs

```diff
@@ -397,6 +397,17 @@ setCache dbName key value = do
       id
   pure $ fromCustomEitherExF fromUnitEx eRes
 
+setCacheWithOpts :: forall st rt. String -> String ->  String -> Maybe Milliseconds -> SetOptions -> BackendFlow st rt (Either Error Boolean)
+setCacheWithOpts dbName key value mbTtl opts = do
+  eRes <- wrap $ RunKVDBEither dbName
+          ((toEitherEx <<< bimap toDBError id) <$> KVDB.setCacheWithOpts key value mbTtl opts)
+          KVDBMock.mkKVDBActionDict
+          (Playback.mkEntryDict
+            ("dbName: " <> dbName <> ", setCache, key: " <> key <> ", value: " <> value <> ", opts: " <> (show opts))
```

Comment:
change "setCache" to "setCacheWithOpts" in both the lines.

Reply:
Updated.